modules/openapi-generator/src/main/resources/JavaSpring/apiUtil.mustache

```diff
@@ -2,7 +2,10 @@ package {{apiPackage}};
 
 {{#reactive}}
 import java.nio.charset.StandardCharsets;
+import org.springframework.core.io.buffer.DefaultDataBuffer;
```

Comment:
should these be wrapper in `{{#reactive}}` block?

Reply:
yea they are only used by the reactive portion of the file